src/ucp/tag/rndv.c

```diff
@@ -385,9 +482,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_get_zcopy, (self),
     state = rndv_req->send.state.dt;
     ucp_dt_iov_copy_uct(iov, &iovcnt, max_iovcnt, &state, rndv_req->send.buffer,
                         ucp_dt_make_contig(1), length);
-
-    lane = rndv_req->send.lane;
-    rkey = ucp_tag_rndv_rkey(rndv_req, 0);
+    iov[0].memh = ucp_tag_rndv_get_lane_info(rndv_req, lane_idx)->memh;
```

Comment:
How about to encapsulate `memh` initialization into `ucp_dt_iov_copy_uct` ?

Reply:
```ucp_dt_iov_copy_uct``` is too general and not designed to take separate memh. original code (before ```ucp_dt_iov_copy_uct``` refactoring) fill single IOV entry and was easy to read, for now we have to post-process ```ucp_dt_iov_copy_uct``` call